src/main/java/edu/nitt/delta/bustracker/controller/api/VehicleController.java

```diff
@@ -117,4 +117,24 @@
                     HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    @PostMapping("/{id}/status")
+    public ResponseEntity<VehicleResponse> changeVehicleStatus(
+            @PathVariable String id, @RequestBody Vehicle vehicle) {
```

Comment:
Having only the boolean value in the request body would be cleaner rather than passing the entire vehicle object, as the client may not always have that object in memory.